pages/templates/pages/microsite_page.html

```diff
@@ -206,5 +227,7 @@
     <div class="microsite">
       {{ block.super }}
     </div>
+  {% else %}
+    {% include "partials/footer-js.html" %}
```

Comment:
Could you please add code comments to explain what is going on here? This is confusing.

Reply:
Is this a solution to the earlier discussion about decoupling footer scripts from footer content? Why can't we do it in more straightforward way?